cluster-install/install.sh

```diff
@@ -2641,8 +2932,9 @@ step_cleanup_tmp_files() {
 #======================================================================
 #== Main
 #======================================================================
+process_args "$@"
 set_log_level
-logln $__INFO "====== Astra Cluster Installer ${__RELEASE_VERSION} ======"
+logln $__INFO "====== Astra Unified Installer ${__RELEASE_VERSION} ======"
```

Comment:
**Astra Unified Installer** has been chosen as the official name of the script by Vijitha/PMs. Related card to deal with the folder name (which requires a change in both this repo and the polaris repo): https://jira.ngage.netapp.com/browse/ASTRACTL-33688